appcenter-data/Data.js

```diff
@@ -5,10 +5,26 @@ const ReactNative = require('react-native');
 
 const { AppCenterReactNativeData } = ReactNative.NativeModules;
 
+const TimeToLive = {
+    INFINITE: -1,
+    NO_CACHE: 0,
+    DEFAULT: 86400
+};
+
+const DefaultPartitions = {
+    USER_DOCUMENTS: 'user',
+    APP_DOCUMENTS: 'readonly'
+};
+
 const Data = {
-    read(documentId, partition) {
-        return AppCenterReactNativeData.read(documentId, partition).then(result => {
+    TimeToLive,
+    DefaultPartitions,
+    read(documentId, partition, ttl) {
```

Comment:
Actually we need ReadOptions as an object like on other SDKs.

Reply:
Added